vcd/resource_vcd_catalog_item_test.go

```diff
@@ -0,0 +1,131 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/go-vcloud-director/govcd"
+	"testing"
+)
+
+var TestAccVcdCatalogItem = "TestAccVcdCatalogItemBasic"
+var TestAccVcdCatalogItemDescription = "TestAccVcdCatalogItemBasicDescription"
+
+func TestAccVcdCatalogItemBasic(t *testing.T) {
+
+	var catalogItem govcd.CatalogItem
+	var params = StringMap{
+		"Org":             testConfig.VCD.Org,
+		"Vdc":             testConfig.VCD.Vdc,
+		"Catalog":         testConfig.VCD.Catalog.Name,
+		"CatalogItemName": TestAccVcdCatalogItem,
+		"Description":     TestAccVcdCatalogItemDescription,
+		"ovaPath":         "../test-resources/test_vapp_template.ova",
```

Comment:
The field names `ovaPath`, `pieceSize`, and `progress` should be capitalized.
Their values should be included in the `testConfig` structure.
I suggest renaming `progress` to `UploadProgress` and `pieceSize` to `UploadChunkSize`.

Reply:
Thank you. Fixed.